spec/services/utilities/employment_income_monthly_equivalent_calculator_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:instance) { described_class.new(employment) }
   let(:assessment) { create :assessment }
   let(:employment) { create :employment, assessment: }
-  let(:payments) { employment.employment_payments.order(:date) }
+  let(:payments) { instance.call }
```

Comment:
I think you can remove the 'before instance.call' line and maybe inline this let - the becomes expect(instance.call) to eq <something> which reads quite nicely